Jeonghyeok/src/boj_01/Main.java

```diff
@@ -0,0 +1,52 @@
+package boj_01;
+
+import java.util.*;
+
+public class Main {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        String input = sc.nextLine();
+        Stack<Character> stack = new Stack<>();
```

Comment:
멘토링 시간에 저희가 스택 문제를 풀 때에는 `Stack` 대신 `Deque`를 사용하는 것을 권장한다고 말씀드렸습니다! 
Stack은 Vector를 상속받아서 단일 스레드에서 불필요한 락이 걸릴 수 있어 비효율적인 면이 있습니다. 
```suggestion
        Deque<Character> stack = new ArrayDeque<>();
```

Reply:
새벽에 생각없이 풀다가 다 풀고나서 아차 했어요..

Reply:
아하 ㅎ 새벽에 올라온거 보긴 했는데 그럴 수 있숩니당..ㅋㅋㅋㅋ 다른건 또 잘 쓰셨어서....